src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -708,21 +731,27 @@ void CommandRequestImpl::AddDissalowedParameterToInfoString(
 void CommandRequestImpl::AddDisallowedParametersToInfo(
     smart_objects::SmartObject& response) const {
   std::string info;
+  uint32_t params_added = 0;
 
   RPCParams::const_iterator it =
       removed_parameters_permissions_.disallowed_params.begin();
   for (; it != removed_parameters_permissions_.disallowed_params.end(); ++it) {
     AddDissalowedParameterToInfoString(info, (*it));
+    ++params_added;
   }
 
   it = removed_parameters_permissions_.undefined_params.begin();
   for (; it != removed_parameters_permissions_.undefined_params.end(); ++it) {
     AddDissalowedParameterToInfoString(info, (*it));
+    ++params_added;
   }
 
   if (!info.empty()) {
-    info += " disallowed by policies.";
-
+    if (params_added > 1) {
+      info += " parameters are disallowed by Policies";
```

Comment:
@AKalinich-Luxoft 
Won't be better something like
```
std::string article = removed_parameters_permissions_.undefined_params > 0 ||
                              removed_parameters_permissions_.disallowed_params > 0 ?
                              " are " : " is ";
info += "parameter(s) " + article + " disallowed by Policies"
```
Logic based on counter looks more error prone.

Reply:
@dev-gh agree. Logic was updated in https://github.com/smartdevicelink/sdl_core/pull/1799/commits/a693891743df516a77f3a65c451f24b5da9aafac